modInterface.py

```diff
@@ -47,12 +48,19 @@ def initWindow(self):
         self.btnAddProduct["width"] = "16"
         self.btnAddProduct.place(x = 615, y = 50)
 
+        self.btnAddProduct = Button(self, text = "Remove Product", command=self.openRemoveProductWindow)
+        self.btnAddProduct["width"] = "16"
+        self.btnAddProduct.place(x = 755, y = 50)
+
```

Comment:
Should this be "btnRemoveProduct"? Looks like we already have an Add Product button.